tests/FSharp.Test.Utilities/XunitHelpers.fs

```diff
@@ -91,6 +91,23 @@ module TestCaseCustomizations =
         else
             testCase.TestMethod
 
+    let sha = Security.Cryptography.SHA256.Create()
+
+    // We add extra trait to each test, of the form "batch=n" where n is between 1 and 4.
+    // It can be used to filter on in multi-agent testing in CI
+    // with dotnet test filter switch, for example "--filter batch=1"
+    // That way each agent can run test for a batch of tests. 
+    let NumberOfBatchesInMultiAgentTesting = 4u
```

Comment:
Hardcoded number of parallel jobs. To make it more flexible and always in sync with the CI, we could read this value from env var set in pipelines.

Reply:
We could spare even more machines. Did you try larger numbers? Are the wins noticable, or is the "next slowest job" a different one already at 4?

Reply:
Yes, `-testVS` is currently the slowest, because it's not parallelized at all.